docs/styleguide.config.js

```diff
@@ -252,67 +253,67 @@ module.exports = {
         '../react/SwipeableDrawer/index.js',
         '../react/TextareaAutosize/index.js',
         '../react/Toolbar/index.js',
-        '../react/Zoom/index.js'
-      ]
-    }
+        '../react/Zoom/index.js',
+      ],
+    },
   ],
   components: '../react/**/*.jsx',
   template: {
     head: {
       meta: [
         {
           name: 'viewport',
-          content: 'width=device-width, initial-scale=1.0'
-        }
+          content: 'width=device-width, initial-scale=1.0',
+        },
       ],
       links: [
         {
           rel: 'stylesheet',
-          href: 'https://fonts.googleapis.com/css?family=Lato:400,700,300'
-        }
-      ]
-    }
+          href: 'https://fonts.googleapis.com/css?family=Lato:400,700,300',
+        },
+      ],
+    },
   },
   styleguideComponents: {
     SectionsRenderer: path.join(__dirname, 'components/SectionsRenderer.jsx'),
     TableOfContentsRenderer: path.join(
       __dirname,
       'components/TableOfContentsRenderer.jsx'
     ),
-    Wrapper: path.join(__dirname, 'components/Wrapper')
+    Wrapper: path.join(__dirname, 'components/Wrapper'),
   },
   theme: {
     fontFamily: {
-      base: 'Lato, sans-serif'
-    }
+      base: 'Lato, sans-serif',
+    },
   },
   webpackConfig: webpackMerge(require('./webpack.config.js'), {
     resolve: {
       alias: {
         'cozy-ui/transpiled/react': path.resolve(__dirname, '../react/'),
-        'cozy-ui': path.join(__dirname, '..')
-      }
-    }
+        'cozy-ui': path.join(__dirname, '..'),
+      },
+    },
   }),
   serverPort: 6161,
   skipComponentsWithoutExample: true,
   styleguideDir: path.resolve(__dirname, '../build/react'),
   require: [
     path.join(__dirname, './style.styl'),
-    path.join(__dirname, './styleguide.setup')
+    path.join(__dirname, './styleguide.setup'),
   ],
   exampleMode: 'collapse',
   usageMode: 'collapse',
   context: {
     utils: path.resolve(__dirname, 'utils'),
     isTesting: path.resolve(__dirname, '../react/helpers/isTesting'),
-    content: path.resolve(__dirname, 'fixtures/content')
+    content: path.resolve(__dirname, 'fixtures/content'),
   },
   styles: {
     Playground: {
       preview: {
-        padding: 0
-      }
-    }
-  }
+        padding: 0,
+      },
+    },
+  },
 }
```

Comment:
can you remove all the "," you added please?

Reply:
Ok then I added this file to lint:js